packages/compass-sidebar/src/components/multiple-connections/connections-navigation.tsx

```diff
@@ -490,19 +515,22 @@ const ConnectionsNavigation: React.FC<ConnectionsNavigationProps> = ({
             expanded={expanded}
           />
         </>
-      ) : (
+      )}
+      {connections.length === 0 && (
         <div className={noDeploymentStyles}>
           <Body data-testid="no-deployments-text">
             You have not connected to any deployments.
```

Comment:
Maybe worth noting that in mms if there are no clusters in the project, we will redirect to the clusters overview page before you even see compass-web rendering, so even though the text here is maybe a bit ill-fitting, real users will never see it anyway